manifests/sip.pp

```diff
@@ -201,6 +205,8 @@
   Array[String[1]]               $transports        = [],
   Optional[String]               $encryption        = '',
   Array[Asterisk::Access]        $access            = [],
+  Optional[Enum['yes', 'no']]    $trustrpid         = undef,
+  Optional[Enum['yes', 'no']]    $sendrpid          = undef
```

Comment:
According to the example configuration file from asterisk, there's another possible value for sendrpid, "pai" (there's also "rpid" but since it's equivalent to "yes" I gues we can omit it to simplify the module).

https://github.com/asterisk/asterisk/blob/master/configs/samples/sip.conf.sample#L369

Would you mind adding 'pai' as one of the options in the enum?

Reply:
I have added both rpid and pai as options